core/integration/client_tests.go

```diff
@@ -17,16 +17,25 @@ package integration
 import (
 	"bytes"
 	"context"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
 	"reflect"
 	"sort"
 	"testing"
 	"time"
 
+	"github.com/google/keytransparency/core/testdata"
+	"github.com/google/keytransparency/impl/authentication"
+	"github.com/google/trillian/types"
+
 	"github.com/google/keytransparency/core/client"
 	"github.com/google/keytransparency/core/sequencer"
 	"github.com/google/keytransparency/core/testutil"
 
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/google/martian/log"
```

Comment:
Should this import be `"log"` or `"glog"`?

Reply:
I suggest glog;)

Reply:
This was a pre existing issue. Fixed in #1197